src/manifold/src/impl.cpp

```diff
@@ -185,7 +186,8 @@ struct CoplanarEdge {
   const int numProp;
   const float precision;
 
-  __host__ __device__ void operator()(
+  // FIXME: race condition
```

Comment:
In theory it can be worse, as this is UB. In practice, it may or may not cause non-determinism depending on whether or not the assigned value is different.

Reply:
I will just put this into a new issue